integration/utils/config.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  baseUrl: 'http://localhost:9000',
+  host: process.env.HOST || 'chrome',
+  sauce: {
```

Comment:
There should be a config for Safari as well here. We can exclude safari from tests if there are compatibility issues but by default we should have the support.

Reply:
Yeah, make sense, I will add Safari in my revision. 👍

Reply:
It should be a separate function to get config for Chrome and Firefox etc. You can add Safari later if we want (We do not run Safari in JS SDK either).

Reply:
Yeah, I can delete this file and make it into a function like how we did in JS SDK. So I won't add Safari for now.

Reply:
updated. check the `configure(host)` function in `DriverFactory`